core/ivr/ivr.go

```diff
@@ -390,9 +384,13 @@ func StartIVRFlow(
 		return fmt.Errorf("no ivr session created")
 	}
 
+	// mark our call as started
+	if err := call.SetInProgress(ctx, rt.DB, sessions[0].UUID(), time.Now()); err != nil {
```

Comment:
bumped until after session is created so we can update our status and session UUID at same time